packages/fields/src/actions/reset-post.tsx

```diff
@@ -1,300 +0,0 @@
-	dispatch( noticesStore ).removeNotice( noticeId );
-	if ( ! isTemplateRevertable( template ) ) {
-		dispatch( noticesStore ).createErrorNotice(
-			__( 'This template is not revertable.' ),
-			{
-				type: 'snackbar',
-			}
-		);
-		return;
-	}
-
-	try {
-		const templateEntityConfig = select( coreStore ).getEntityConfig(
-			'postType',
-			template.type
-		);
-
-		if ( ! templateEntityConfig ) {
-			dispatch( noticesStore ).createErrorNotice(
-				__(
-					'The editor has encountered an unexpected error. Please reload.'
-				),
-				{ type: 'snackbar' }
-			);
-			return;
-		}
-
-		const fileTemplatePath = addQueryArgs(
-			`${ templateEntityConfig.baseURL }/${ template.id }`,
-			{ context: 'edit', source: template.origin }
-		);
-
-		const fileTemplate = ( await apiFetch( {
-			path: fileTemplatePath,
-		} ) ) as any;
-		if ( ! fileTemplate ) {
-			dispatch( noticesStore ).createErrorNotice(
-				__(
-					'The editor has encountered an unexpected error. Please reload.'
-				),
-				{ type: 'snackbar' }
-			);
-			return;
-		}
-
-		const serializeBlocks = ( { blocks: blocksForSerialization = [] } ) =>
-			__unstableSerializeAndClean( blocksForSerialization );
-
-		const edited = select( coreStore ).getEditedEntityRecord(
-			'postType',
-			template.type,
-			template.id
-		) as any;
-
-		// We are fixing up the undo level here to make sure we can undo
-		// the revert in the header toolbar correctly.
-		dispatch( coreStore ).editEntityRecord(
-			'postType',
-			template.type,
-			template.id,
-			{
-				content: serializeBlocks, // Required to make the `undo` behave correctly.
-				blocks: edited.blocks, // Required to revert the blocks in the editor.
-				source: 'custom', // required to avoid turning the editor into a dirty state
-			},
-			{
-				undoIgnore: true, // Required to merge this edit with the last undo level.
-			}
-		);
-
-		const blocks = parse( fileTemplate?.content?.raw );
-
-		dispatch( coreStore ).editEntityRecord(
-			'postType',
-			template.type,
-			fileTemplate.id,
-			{
-				content: serializeBlocks,
-				blocks,
-				source: 'theme',
-			}
-		);
-
-		if ( allowUndo ) {
-			const undoRevert = () => {
-				dispatch( coreStore ).editEntityRecord(
-					'postType',
-					template.type,
-					edited.id,
-					{
-						content: serializeBlocks,
-						blocks: edited.blocks,
-						source: 'custom',
-					}
-				);
-			};
-
-			dispatch( noticesStore ).createSuccessNotice(
-				__( 'Template reset.' ),
-				{
-					type: 'snackbar',
-					id: noticeId,
-					actions: [
-						{
-							label: __( 'Undo' ),
-							onClick: undoRevert,
-						},
-					],
-				}
-			);
-		}
-	} catch ( error: any ) {
-		const errorMessage =
-			error.message && error.code !== 'unknown_error'
-				? error.message
-				: __( 'Template revert failed. Please reload.' );
-
-		dispatch( noticesStore ).createErrorNotice( errorMessage, {
-			type: 'snackbar',
-		} );
-	}
-};
-
-const resetPostAction: Action< Template | TemplatePart > = {
```

Comment:
This action seems still necessary for template parts no?

Reply:
Hm, I guess! Unless we want to adjust them in this PR too haha :D